libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -141,28 +142,25 @@ internal class MapboxTripSession(
                 }
             }
         }
-        logD(
-            "routes update (reason: ${setRoutesInfo.reason}, " +
-                "route IDs: ${routes.map { it.id }}) - finished",
-            LOG_CATEGORY
-        )
+        logD(LOG_CATEGORY, logMessage("- finished"))
```

Comment:
Why can't you just create a String?

Reply:
There might be both. Why do you think one is better than another?

Reply:
Why create a lambda that creates a string when you can just create a string? It will be created anyway so I see adding a lambda as an unnecessary complication.

Reply:
It's a micro-performance (lazy) upgrade, I would keep it as is. Don't see a real matter to change to the field.

Reply:
Oh, I didn't notice you were reusing it. OK then.